### ISSUE TITLE
Rendering of Latex/Mathjax in print output of pages

### ISSUE THREAD

**Issue by juststormy** [elmsln/elmsln#903]
I was showing a faculty member how our pages render mathjax/latex. He had interest in student being able to print out the content for assistance review. However, when you go to print the page, it just renders the code for the latex/mathjax and does not compile it into the actual formulas. I tested this in gitbook as well. Same things happens. Is there a way to keep the equations intact when we convert the content to pdf or word?

**Comment by btopro** [comment]
To clarify you are hitting the print page link then printing the page right?

**Comment by btopro** [comment]
this should be fixed, had to do with javascript not processing the mathjax (cause it wasn't included). Now it should be included correctly

**Comment by btopro** [comment]
yup, just proved this fixes it.
